configuration/scripts/machines/env.gaea_intel

```diff
@@ -7,20 +7,27 @@ endif
 
```

Comment:
The env.gaea should probably be env.gaeac5 since this I presume gaeac6 would be different.